fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/metadata/SyntheticRecordType.java

```diff
@@ -89,7 +90,13 @@ public boolean isSynthetic() {
 
     @API(API.Status.INTERNAL)
     @Nonnull
-    public abstract CompletableFuture<FDBSyntheticRecord> loadByPrimaryKeyAsync(FDBRecordStore store, Tuple primaryKey);
+    public CompletableFuture<FDBSyntheticRecord> loadByPrimaryKeyAsync(FDBRecordStore store, Tuple primaryKey) {
+        return loadByPrimaryKeyAsync(store, primaryKey, IndexOrphanBehavior.ERROR);
+    }
+
+    @API(API.Status.INTERNAL)
+    @Nonnull
+    public abstract CompletableFuture<FDBSyntheticRecord> loadByPrimaryKeyAsync(FDBRecordStore store, Tuple primaryKey, IndexOrphanBehavior orphanBehavior);
```

Comment:
I'm not sure what exactly we should do with `INTERNAL`, but I think the PR description should probably call out that `SyntheticRecordType` has changed the behavior for `loadByPrimaryKeyAsync`, in case anyone is extending it.

Reply:
Done (changed PR description).